.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
```

Comment:
I had to create `.circleci/config.yml` because CircleCI removed support for 1.0 version on August 31, 2018